tests/Stripe/AccountTest.php

```diff
@@ -179,4 +179,92 @@ public function testCanCreateLoginLink()
         $resource = Account::createLoginLink(self::TEST_RESOURCE_ID);
         $this->assertInstanceOf("Stripe\\LoginLink", $resource);
     }
+
+    public function testSerializeNewAdditionalOwners()
```

Comment:
Thanks for adding all these tests. Account's serialization is problematic enough that you really can't have enough test coverage of it.